### ISSUE TITLE
Feature Request: Simple text menu by talking to the bot account directly.

### ISSUE THREAD

**Issue by TomTheDragon** [42wim/matterbridge#183]
### Which version of matterbridge are you using?
0.12.1

### Desciption of the request
I am not sure if this is possible but i thought it would be cool to have some sort of menu you could use, by talking to the bot directly. You then could get some extra informations like a printout of the userlists from the the other connected chatrooms.

For example:
After the bot sees any kind of private text input, it prints out an list with commands. You can then enter something like /userlist to get an printout of the userlist(s) of the connected chatroom(s).

**Comment by 42wim** [comment]
Yes, something I'm also thinking about, but by using the rest api. 
But I don't have the time for this at the moment.
